<launch>

  <!-- Set the joystick parameter --> 
  <param name="joy_node/dev" value="/dev/input/js2" />
  
  <!-- Set joystick mapping parameters -->
  <param name="joy_reporter_node/speed_axis"  value="4" />
  <param name="joy_reporter_node/steer_axis"  value="0" />
  <param name="joy_reporter_node/spd_up_btn"  value="3" />
  <param name="joy_reporter_node/spd_dwn_btn" value="0" />
  <param name="joy_reporter_node/yaw_up_btn"  value="1" />
  <param name="joy_reporter_node/yaw_dwn_btn" value="2" />      
  
  <!-- Launch the joy node -->
  <node pkg="joy" type="joy_node" name="joy_node" output="screen" />
   
  <!-- Launch joystick reporter node -->
  <node pkg="hw05_crist" type="joy_reporter.py"
  	name="joy_reporter_node" output="screen" />


</launch>
